builder/dockerfile/parser/parser.go

```diff
@@ -177,7 +177,7 @@ func (d *Directive) possibleParserDirective(line string) error {
 func NewDefaultDirective() *Directive {
 	directive := Directive{}
 	directive.setEscapeToken(string(DefaultEscapeToken))
-	directive.setPlatformToken(runtime.GOOS)
+	directive.setPlatformToken(DefaultPlatformToken)
```

Comment:
I noticed `DefaultPlatformToken` is only used in this file; can you un-export it?